docs/content.zh/docs/dev/table/common.md

```diff
@@ -387,19 +387,19 @@ Table table = ...;
 
 // register the view named 'exampleView' in the catalog named 'custom_catalog'
 // in the database named 'custom_database' 
-tableEnv.createTemporaryView("exampleView", table);
```

Comment:
I notice that the English version uses tableEnv as the variable name. I suggest changing them both to be consistent in this PR. Changing the variable definition to tableEnv here - would seem to be the minimal change - unless there is a compelling reason to use tEnv.